workspace/__lib__/xod/core/discretize/patch.xodp

```diff
@@ -0,0 +1,155 @@
+{
+  "description": "Rounds input `X` up to a nearest threshold — `T0`, `T1`, etc. If `X` is even greater than last threshold, the value is still round to it. Assumes thresholds are in ascending order: `T0` < `T1` etc",
```

Comment:
Rounds input `X` up to **the first threshold `T0`, `T1`, ..., `T𝑛` greater or equal to the `X`**. If `X` is even greater than **the** last threshold, the value is ~still~ round **down** to it. ~Assumes thresholds are in ascending order: `T0` < `T1` etc~

With changes applied:

Rounds input `X` up to the first threshold `T0`, `T1`, ..., `T𝑛` greater or equal to the `X`. If `X` is even greater than the last threshold, the value is round down to it.